apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/GraphQLCompiler.kt

```diff
@@ -45,7 +45,10 @@ class GraphQLCompiler(val logger: Logger = NoOpLogger) {
     )
 
     val files = args.graphqlFiles
-    checkDuplicateFiles(roots, files)
+    /**
+     * Android can have duplicate files but that's ok as long as they do not define the same operations (which is checked in IRBuilder)
+     */
+    // checkDuplicateFiles(roots, files)
```

Comment:
Shouldn't we just delete this completely now?

Reply:
I guess I got sentimentally attached to this method 😂 . You're right, I'll remove it.